Terraform/README.md

```diff
@@ -0,0 +1,41 @@
+# Action Samples for deploying using Terraform
+
+With Terraform workflows, you can automate your terraform templates to deploy to Azure.
+
+Terraform templates use Azure Resource Manager to deploy resources to Azure. More details on Terraform Provider for Azure can be found [here](https://www.terraform.io/docs/providers/azurerm/index.html).
+
+## Configure Azure credentials
+
+To fetch the credentials required to authenticate with Azure, run the following command:
+
+```sh
+az ad sp create-for-rbac --name "myApp" --role contributor \
+                            --scopes /subscriptions/{subscription-id}/resourceGroups/{resource-group} \
+                            --sdk-auth
+
+  # Replace {subscription-id}, {resource-group} with the subscription, resource group details
+
+  # The command should output a JSON object similar to the example below
+
+  {
+    "clientId": "<GUID>",
+    "clientSecret": "<GUID>",
+    "subscriptionId": "<GUID>",
+    "tenantId": "<GUID>",
+    (...)
+  }
+```
+
+Add the JSON output as secrets TF_VAR_agent_client_id, TF_VAR_agent_client_secret, TF_VAR_subscription_id, TF_VAR_tenant_id in the GitHub repository. For steps to create and storing secrets, please check [here](https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets)
+
+Store the Terraform templates in a directory and update the value of `TF_WORKING_DIR` in the environment for deployment.
```

Comment:
I believe this comment is no longer relevant. You can instead call out that the action expects the terraform templates to be in the root. If user would like to have the templates stored in a different directory, they can enhance it by providing appropriate path to the action input.